app/models/offender.rb

```diff
@@ -50,7 +48,7 @@ def handover_progress_task_completion_data
 
   # Returns the most recent parole record (can be a future parole application), regardless of activity status and outcome.
   def most_recent_parole_review
-    filtered_parole_reviews.max_by(&:sortable_date)
+    @most_recent_parole_review ||= parole_reviews.ordered_by_sortable_date.to_a.last
```

Comment:
How come you to_a the collection?

Reply:
`.last` directly on the association was causing an error

Reply:
was it ActiveRecord::IrreversibleOrderError?